android_app/app/src/main/java/com/health/openscale/core/bluetooth/BluetoothStandardWeightProfile.java

```diff
@@ -823,29 +805,59 @@ protected void chooseExistingScaleUser(Vector<ScaleUser> userList) {
         chooseScaleUserUi(choices);
     }
 
-    protected String getInitials(String fullName) {
-        if (fullName == null || fullName.isEmpty() || fullName.chars().allMatch(Character::isWhitespace)) {
-            return getDefaultInitials();
-        }
-        return buildInitialsStringFrom(fullName).toUpperCase();
+    protected String getInitials(final String fullName, final int size) {
+        if (0 == size)
+            return StringUtils.EMPTY_STRING;
```

Comment:
Side note: I'm not professional Java developer - openScale is only second project where I have to use JAVA. Take everything I say about Java with grain of salt.

As far as I understand how JAVA works - it will create only  one constant empty string in whole project and whenever you use `""` it will be re-used. Therefore form performance point of view it adds no benefit - neither compile nor runtime. But again - I'm not Java expert, my understanding might be incorrect.

Reply:
🥇 to use `org.apache.commons.lang3.StringUtils` 's `EMPTY` constant